### PR TITLE
Math magicians: Full Website

### PR DESCRIPTION
**Pull request by Johnadibe**
## Math Magicians: Full Website

- Created the full website that consisted of 3 pages: Home, Calculator, and Quote.
- Matched the general layout with the provided wireframes.
- Added my own styles to improve the look and feel.
- Created a <Route> for every page.
- Ensured that navigation links for all 3 routes are displayed on each page.
- Used React Router to make the routes work.

### CODE REVIEW

File: src/components/pages/Home.jsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+/* eslint-disable react/jsx-indent */
+/* eslint-disable max-len */
+/* eslint-disable no-trailing-spaces */
```
Comment (binhussen) [inline src/components/pages/Home.jsx]: - Kindly, you disabled some functionality of the linter, :warning: the linter Lint checks against best practices, code style guideline violations, and structural problems. so please delete line number 1-3 :+1:

File: src/components/pages/Quote.jsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
+/* eslint-disable react/jsx-indent */
+/* eslint-disable max-len */
+/* eslint-disable no-trailing-spaces */
```
Comment (binhussen) [inline src/components/pages/Quote.jsx]: - Kindly, you disabled some functionality of the linter, :warning: the linter Lint checks against best practices, code style guideline violations, and structural problems. so please delete line number 1-3 :+1:

File: src/index.js [hunk 3]
```diff
@@ -1,10 +1,14 @@
+/* eslint-disable import/no-extraneous-dependencies */
```
Comment (binhussen) [inline src/index.js]: - Kindly, you disabled some functionality of the linter, :warning: the linter Lint checks against best practices, code style guideline violations, and structural problems. so please delete line number 1 :+1:

File: src/components/Navbar.jsx [hunk 4]
```diff
@@ -0,0 +1,36 @@
+/* eslint-disable import/no-extraneous-dependencies */
+/* eslint-disable no-trailing-spaces */
```
Comment (binhussen) [inline src/components/Navbar.jsx]: - Kindly, you disabled some functionality of the linter, :warning: the linter Lint checks against best practices, code style guideline violations, and structural problems. so please delete line number 1-2 :+1:

File: src/App.js [hunk 5]
```diff
@@ -1,10 +1,20 @@
+/* eslint-disable import/no-extraneous-dependencies */
```
Comment (binhussen) [inline src/App.js]: - Kindly, you disabled some functionality of the linter, :warning: the linter Lint checks against best practices, code style guideline violations, and structural problems. so please delete line number 1-3 :+1:

**Review by binhussen — CHANGES_REQUESTED** [review]
Hi @Johnadibe  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- All linter checks passed ✔️
- Written Clean Code ✔️
- Used Git flow ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by codecaiine — APPROVED** [review]
Hi @Johnadibe  , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linters errors :heavy_check_mark: 
- Pages work as expected :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**